src/Pecee/Http/Service/Tinder.php

```diff
@@ -77,12 +81,16 @@ public function like($userId) {
         return $this->api('like/' . $userId);
     }
 
+	public function superLike($userId) {
+		return $this->api('like/' . $userId .'/super/');
+	}
+
     public function pass($userId) {
         return $this->api('pass/' . $userId);
     }
 
-    public function updates() {
-        return $this->api('updates');
+    public function updates($lastActivityTime = '') {
```

Comment:
updates now its a POST method that require last_activity_date. If last_activity_date its not passed its going to show all the messages and matches.